.gitmodules

```
@@ -2,6 +2,7 @@
 #
 # SPDX-License-Identifier: MPL-2.0
 
-[submodule "power-grid-model"]
-	path = power-grid-model
-	url = git@github.com:PowerGridModel/power-grid-model.git
+[submodule "deps/power-grid-model"]
+	path = deps/power-grid-model
+	url = https://github.com/PowerGridModel/power-grid-model.git
```

On a second thought, please update this line to use the ssh protocal
```suggestion
	url = git@github.com:PowerGridModel/power-grid-model.git
```

---

not all our users may have SSH installed. In addition, it is and open-source, so security is also not an issue on that side.

The HTPPS protocol is actually probably desired...

---

I totally agree on the esay to use side of https. We can change it back in the next PR. I was just under the impression that open source project tend to prefer ssh because it is *cool*

---

🤣